CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Fixed a header name conflict with Realm (Issue #90)
 * Incorrect passwords in RAR5 archives now produce a specific and helpful error message (PR #101 - Thanks to [@gpotari](https://github.com/gpotari) for the idea and implementation!)
 * Updated to v6.1.7 of UnRAR library (PR #103 - Thanks to [@gpotari](https://github.com/gpotari) )
+* Fixed compile error for Example project. Added missing CFBundleShortVersionString key to Info.plist
```

Comment:
Please update the CHANGELOG to reflect the actual nature of the problem being fixed (a crash, not a compilation error).

```suggestion
* Fix crash on start of Example app (PR #107 - Thanks to [@gpotari](https://github.com/gpotari))
```